ujt/utils.py

```diff
@@ -133,3 +135,48 @@ def get_latest_tapped_element(
         return tap_node
 
     return tap_node if tap_node["timeStamp"] > tap_edge["timeStamp"] else tap_edge
+
+
+def get_change_over_time_class_from_composite_slis(
+    before_composite_sli: Optional[SLI], after_composite_sli: Optional[SLI]
+) -> str:
+    """Returns the appropriate coloring/gradient based on the composite SLIs.
+
+    We denote parts of a full styling class as a subclass.
+    These include STATUS_HEALTHY, IMPROVED, etc.
+    We join the subclasses together with underscore as a delimiter to create a full class name,
+    that has an assoiated style.
+
+    Args:
+        before_composite_sli: A composite SLI holding the average value of the SLIs over the former half of a time interval.
+        after_composite_sli: A composite SLI holding the average value of the SLIs over the latter half of a time interval.
+
+    Returns:
+        A class name to be added to the element.
+    """
```

Comment:
I placed this function in utils. It feels wrong, but I can't find a better place for it. I suppose this is a downside of the converters/transformers organization convention, since this helper function doesn't really fit into those molds. If we organized by "type of object being acted on" or by feature, this function would be more easily organized.